weekly-updates/Kirstie-Whitaker/Weekly-meetings.md

```diff
@@ -17,6 +18,102 @@ Notes from Kirstie's personal version of the "weekly meetings" file she asks all
 
 ---
 
+## Date: 21 February 2021
+
+### What brought you joy this week?
+
+* My greatest joy this week was spending 3 hours working with AL on the community manager job descriptions.
+  * Huge huge shout out to Brene Brown's values list and the exercise of picking just 2 or 3.
+    It's such a hard task but SO valuable!
+    * https://daretolead.brenebrown.com/workbook-art-pics-glossary/
+    * For community managers I picked **stewardship**, **compassion** and **collaboration**.
+    * For RAMs 🐏 AM and I picked **adaptability**, **legacy** and **perseverance**.
+    * For my job I picked **vision**, **faith** and **justice**.
+  * It surprised me how emotional I was when we started this exercise.
+    It was so valuable to be able to articulate why I was so blocked - I felt incredibly lonely - and what I needed to move forwards - a safe space to be able to explore 🙏💕
+* Coffee and (virutal) hangout with BM.
+  What a delightful man.
+  I miss getting to spend more time with him, and benefiting from his mentorship.
+* I skipped doing the work I was supposed to catch up on Friday to attend the Engage at Turing student talks and they did SUCH a good job.
+  I'm really glad I came along to ask questions and learn a little more.
+  Definitely got a few folks to follow up with 💪
+* Huge thank you to CH who gave me a call after the grant panel interview and said I did a good job.
+  * Means so much 💜
+* Brilliant catch up with IS, absolute joy as usual 🌺🌺🌺
+* IKG was a fantastic support, as usual.
+  * But particularly this week on a Friday during his holiday.
+* Hangout with AllyM - I'm so glad we have this in the calendar now.
+* Really glad SM was able to come to the Turing Way collaboration cafe!! ☕
+* LOVE the help that IS was able to provide for ME! :heart_eyes:
+  * https://github.com/WhitakerLab/WhitakerLabProjectManagement/blob/master/weekly-updates/Maria-Eriksson/Mentoring-IS.md#date-11-february-2021
+* Knowledge share this week made me a bit teary to actually see the work progressing, and hear how important it still is for clinicians.
+* AB sent the TPS newsletter in its new format!
+  * So beautiful!
+  * http://elink.turing.ac.uk/m/1/59207972/p1-b21050-e368bf4a814a4324a28ac2cb84e12511/1/13/843f7bf5-d1bc-4395-b06f-13236425037e
+* Maybe not exactly joy, but I helped a few friends through difficult frustrations with unsupportive colleagues (not at Turing) this week.
+  * I'm glad I could be of some help.
+  * Also, its motherlode dude.
```

Comment:
I never knew that! ✨

Reply:
It's actually supposed to be 2 words.... fixed now. And a link to https://en.wikipedia.org/wiki/Mother_lode